src/test/java/TopNavigationTest.java

```diff
@@ -0,0 +1,150 @@
+    }
+
+    @DataProvider
+    public Object[][] thirdCategoriesLevelWithExpectedNamesAndLinksWomenTops() {
+        secondLevelTopCategoriesList = topNavigation.getSecondLevelTopCategoriesList();
+
+        return new Object[][]{
+                {secondLevelTopCategoriesList.get(0), List.of("Jackets", "Hoodies & Sweatshirts", "Tees", "Bras & Tanks"), List.of("women/tops-women/jackets-women.html",
+                        "women/tops-women/hoodies-and-sweatshirts-women.html", "women/tops-women/tees-women.html", "women/tops-women/tanks-women.html")},
+                //{secondLevelTopCategoriesList.get(1), List.of("Pants", "Shorts"), List.of("women/bottoms-women/pants-women.html", "women/bottoms-women/shorts-women.html")},
+                //{secondLevelTopCategoriesList.get(0), List.of("Jackets", "Hoodies & Sweatshirts", "Tees", "Tanks"), List.of("men/tops-men/jackets-men.html",
+                       // "men/tops-men/hoodies-and-sweatshirts-men.html", "men/tops-men/tees-men.html", "men/tops-men/tanks-men.html")},
+                //{secondLevelTopCategoriesList.get(1), List.of("Pants", "Shorts"), List.of("men/bottoms-men/pants-men.html", "men/bottoms-men/shorts-men.html")},
+        };
+    }
+
+    @Test(dataProvider = "thirdCategoriesLevelWithExpectedNamesAndLinksWomenTops")
+    public void topNavigation_ThirdLevelCategoriesWomenTops_CorrectlyDisplayed(List<String> expectedCategoriesNames, List<String> expectedCategoriesLinks) {
+        //when
+        List<WebElement> thirdLevelCategories = topNavigation.hoverOnCategorySecondLevel();
+        
+        //then
+        validateMenuCategoriesNamesAndLinksForThirdLevel(thirdLevelCategories, expectedCategoriesNames, expectedCategoriesLinks);
+    }
+
+    public void validateMenuCategoriesNamesAndLinks(List<WebElement> categoriesList, List<String> expectedCategoriesNames,
+                                                    List<String> expectedCategoriesLinks) {
+
+        List<String> actualCategoriesNames = new ArrayList<>();
+        List<String> actualCategoriesLinks = new ArrayList<>();
+        List<String> expectedCategoriesLinksWithDomain = new ArrayList<>();
+
+        for (WebElement category : categoriesList) {
+            WebElement categoryName = category.findElement(topNavigationBarCategoryNameFirstLevel);
+            Assertions.assertThat(categoryName.isDisplayed()).isTrue();
+            actualCategoriesNames.add(categoryName.getText());
+            WebElement categoryLink = category.findElement(topNavigationBarCategoryFirstLevelLink);
+            Assertions.assertThat(categoryLink.isEnabled()).isTrue();
+            actualCategoriesLinks.add(categoryLink.getAttribute(Attribute.HREF.name()));
+        }
+
+        for (String link : expectedCategoriesLinks) {
+            expectedCategoriesLinksWithDomain.add("https://magento.softwaretestingboard.com/" + link);
```

Comment:
HOMEPAGE_URL should be used instead of "https://magento.softwaretestingboard.com/"